packages/pouchdb/README.md

```diff
@@ -0,0 +1,37 @@
+# PouchDB Adapter
```

Comment:
Dig all the work you put into updating the READMEs, package.json, etc. as well!

Small nit though. Lets make this header consistent with the other ones. Something like:

```markdown
<p align="center">
   <br/>
   <a href="https://next-auth.js.org" target="_blank">
   <img height="64px" src="https://next-auth.js.org/img/logo/logo-sm.png" /></a>&nbsp;&nbsp;&nbsp;&nbsp;<img height="64px" src="https://raw.githubusercontent.com/nextauthjs/adapters/canary/packages/pouchdb/logo.svg" />
   <h3 align="center"><b>PouchDB Adapter</b> - NextAuth.js</h3>
   <p align="center">
   Open Source. Full Stack. Own Your Data.
   </p>
   <p align="center" style="align: center;">
      <img src="https://github.com/nextauthjs/adapters/actions/workflows/canary.yml/badge.svg" alt="Build Test" />
      <img src="https://img.shields.io/bundlephobia/minzip/@next-auth/pouchdb-adapter" alt="Bundle Size"/>
      <img src="https://img.shields.io/npm/v/@next-auth/pouchdb-adapter" alt="@next-auth/pouchdb-adapter Version" />
   </p>
</p>

## Overview
```

See [fauna/readme.md](https://raw.githubusercontent.com/nextauthjs/adapters/canary/packages/fauna/README.md) for example.

Reply:
I updated the doc according to the format of the other adapters docs.